src/js/planting.js

```diff
@@ -68,7 +68,7 @@ export default class extends EventEmitter {
       collection: this.session().objects(),
       app: this,
     });
-    this.toolbox = new Toolbox.View.Sidebar({
+    this.toolbox = new Sidebar({
```

Comment:
If we're moving away from `Toolbox` name for model attribute name should be changes also?